buf/registry/owner/v1beta1/user.proto

```diff
@@ -54,20 +54,15 @@ message User {
     (buf.validate.field).required = true,
     (buf.validate.field).enum.defined_only = true
   ];
-  // The role that the User has at the BSR instance level.
-  UserServerRole server_role = 7 [
```

Comment:
Say we do keep this, should this be:

- hidden: display unspecified to all users, except if you're logged in and are a server admin
- masked: always display "member", unless logged in and are a server admin
- always show correct server role if they are authenticated
 
And the last option is,

- always show the server role, regardless if authenticated or anonymous

Reply:
If we can get away with not needing it for now, I think you're right that that is optimal.